ShenkinStore/Controllers/ProductsController.cs

```diff
@@ -172,5 +175,21 @@ public async Task<IActionResult> Wallets()
         {
             return View(await _context.Products.ToListAsync());
         }
-    }
+
+        public void facebook(string ProductName, decimal ProductPrice, string ProductImage)  //after we add aflight we posted in facebool
```

Comment:
please change to a more meaningful name (createFacebookPost or postProductOnFacebookPage)

Reply:
Delete the comment